asp.in

```diff
@@ -141,6 +141,10 @@ initialize() {
       if [[ $url = *'git.archlinux.org'* ]]; then
         git remote set-url "$remote" "${url%%:*}://github.com/archlinux/svntogit-$remote.git"
       fi
+      # https://github.blog/2021-09-01-improving-git-protocol-security-github/
+      if [[ $url = *'git://github.com'* ]]; then
```

Comment:
We also need to handle the case where the user hasn't migrated from git.archlinux.org yet (not tested):
```suggestion
      # https://github.blog/2021-09-01-improving-git-protocol-security-github/
      if [[ $url = *'git.archlinux.org'* ]] || [[ $url = *'git://github.com'* ]]; then
        git remote set-url "$remote" "https://github.com/archlinux/svntogit-$remote.git"
      fi
```

Reply:
That's already handled by the if statement above

Reply:
Yes, but you need to run `asp` twice.